### PR TITLE
Add doc for ST_ISOVist

### PR DESCRIPTION
**Pull request by gpetit**
Add doc for ST_ISOVist

### CODE REVIEW

File: docs/dev/ST_ISOVist.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+`Obstacles` can be `LINESTRING(S)` ,`POLYGON(S)`  or `GEOMETRYCOLLECTION(S)`. 
+
+Resulting polygon will be enclosed by a circle defined by the maximum distance (`maxDistance`) parameter.
+
+##### Parameters
+
+* `point` : Point coordinates (x, y, (z)) from which the visibility will be calculated,
+* `obstacles` : Geometry or set of geometries grouped into a simple or complex geometry, used as obstacles,
+* `maxDistance` : Only obstacles located within this distance from the `point` are considered in the calculation *(unit depends on your spatial ref)*,
```
Comment (ebocher) [inline docs/dev/ST_ISOVist.md]: Sure ??? or only Cartesian computation ?

File: docs/dev/ST_ISOVist.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+* `angleStart` : Starting angle from which visibility will be calculated *(exprimed in radian)*,
```
Comment (ebocher) [inline docs/dev/ST_ISOVist.md]: from north ?

File: docs/dev/ST_ISOVist.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+	((70.3 67.2 0, 76.3 51.9 0, 79.2 53.1 0, 81.9 46.1 0, 75.3 43.6 0, 72.6 50.5 0, 68 48.6 0, 61.9 63.9 0, 70.3 67.2 0)), 
+	((68 33.6 0, 64.8 32.5 0, 62.3 39 0, 69.1 41.4 0, 70.7 34.8 0, 68 33.6 0)), 
+	((65.4 31.2 0, 69 21.9 0, 55.1 16.3 0, 51.5 25.7 0, 65.4 31.2 0)), 
+	((59.6 85.6 0, 63.4 75.6 0, 54.8 72.5 0, 51.1 82.2 0, 59.6 85.6 0)), 
+	((62 38.3 0, 62.9 36.1 0, 58.8 34.5 0, 57.9 36.7 0, 62 38.3 0))), 
+	MULTILINESTRING ((55.8 87.4 0, 32.5 78 0), (32.5 78 0, 30.4 82.9 0), (45.7 48.2 0, 38.7 63.1 0), 
+	(13.8 77.7 0, 17.7 67.4 0), (14.9 48.4 0, 9.6 46.6 0), (9.6 46.6 0, 12.1 39.1 0)))') as the_geom;
+{% endhighlight %}
+
+#### Case with just a distance
```
Comment (ebocher) [inline docs/dev/ST_ISOVist.md]: Example with a distance

File: docs/dev/ST_ISOVist.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+CREATE TABLE isovist AS 
+  SELECT ST_ISOVIST(a.the_geom, b.the_geom, 50, 0, 5) as the_geom 
+  FROM point a, polygon b;
+{% endhighlight %}
+
+<img class="displayed" src="../st_isovist_p1_50m_0_to_5.png"/>
+
+-----
+
+#### Showcase : Compute visibilty along a path
```
Comment (ebocher) [inline docs/dev/ST_ISOVist.md]: Application

File: docs/dev/ST_ISOVist.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+* `for(int i=1;i<=256;i+=1)` &rarr; `256` correspond to the maximum number of image produced *(so here the maximum number of point used to compute the visibility along the `path`)*,
+* `ST_Densify(the_geom, 10)` &rarr; `10` is the distance *(exprimed in meter)* used to densify the `path` &rarr; every `10`m a point will be added,
+* `path` is the layer name of the linestring used to computes visibilities,
+* `ST_Accum(the_geom), 100)` &rarr; `100` is the maximum distance used to compute the visibilty *(see `maxDistance` parameter)*,
+* `buildings` is the layer name in which obstacles are stored,
+* `/myUrl/+"test"+i+".png"` &rarr; .png files, prefixed by `test`; will be saved into a folder defined by `/myUrl/`.
+* `src` and `iso` are respectively the output table names for the point where the visibility is calculated and the resulting visibility polygon. 
+
+
+##### 3- Convert all the images into a gif
```
Comment (ebocher) [inline docs/dev/ST_ISOVist.md]: animated gif ?

File: docs/dev/ST_ISOVist.md (hunk 4 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: Ok

File: docs/dev/ST_ISOVist.md (hunk 5 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: ok

File: docs/dev/ST_ISOVist.md (hunk 3 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: ok

File: docs/dev/ST_ISOVist.md (hunk 2 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: 0 rad is oriented to east

File: docs/dev/ST_ISOVist.md (hunk 1 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: Don't understand. 
This text is extracted / adapted from the source code

Comment (nicolas-f) [reply docs/dev/ST_ISOVist.md]: Yes only Cartesian distance

File: docs/dev/ST_ISOVist.md (hunk 2 above)
Comment (nicolas-f) [reply docs/dev/ST_ISOVist.md]: Its in radian, then trigonometric circle

File: docs/dev/ST_ISOVist.md (hunk 1 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: Ok I will add this info

File: docs/dev/ST_ISOVist.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+CREATE TABLE isovist AS 
+  SELECT ST_ISOVIST(a.the_geom, b.the_geom, 50, 0, 5) as the_geom 
+  FROM point a, polygon b;
+{% endhighlight %}
+
+<img class="displayed" src="../st_isovist_p1_50m_0_to_5.png"/>
+
+-----
+
+#### Application : Compute visibilty along a path
```
Comment (nicolas-f) [inline docs/dev/ST_ISOVist.md]: #### Application : Compute visibility along a path

**Comment by gpetit** [comment]
Thx @nicolas-f but here it's only if you want to exprime angles in degree. I guess people are able to convert by themselves.
Any way I will add this : 
```
angleStart : Starting angle from which visibility will be calculated (exprimed in radian - to convert into degree, use the H2 database PI() function),
```

File: docs/dev/ST_ISOVist.md (hunk 6 above)
Comment (gpetit) [reply docs/dev/ST_ISOVist.md]: thx

**Comment by nicolas-f** [comment]
> Thx @nicolas-f but here it's only if you want to exprime angles in degree. I guess people are able to convert by themselves.
> Any way I will add this :
> 
> ```
> angleStart : Starting angle from which visibility will be calculated (exprimed in radian - to convert into degree, use the H2 database PI() function),
> ```

Its not to express in degree here. Its for clarity. If I wanted to express it in degrees then I would have written (value_in_degree / pi) * 180.

**Pull request merged**